scripts/create-database.js

```diff
@@ -0,0 +1,57 @@
+// scripts/create-database.js
+
+// require the pg package
```

Comment:
Comments like this are not really helpful, they just add noise to the file. Comments should really be used to explain _why_ something is being done the way it is and not _how_ (ideally, the code itself should be clear enough to not need any further explanation).